apps/tmux/tmux.py

```diff
@@ -3,8 +3,17 @@
 ctx = Context()
 mod = Module()
 
+apps = mod.apps
+
+apps.tmux = "app.name: tmux"
+
+mod.apps.tmux = """
+tag: terminal
+and win.title: /tmux/
```

Comment:
I don't get tmux in the title when I run it on my machine. I'm also not sure the app.name thing will work because Talon will probably just pick up the name of the terminal.

Probably just want to do:
```
mod.apps.tmux = """
tag: terminal
and tag: user.tmux
"""

Reply:
I have the below in my `tmux` configuration which allowed me to see it in my window title.

```tmux.conf
# Window manager title settings
set-option -g set-titles on
set-option -g set-titles-string "tmux - #{session_name} / #{window_name}: #{pane_title}"
```